script/convert-grammars

```diff
@@ -185,7 +196,7 @@ def load_grammars(tmp_dir, source, all_scopes)
 
   p.fetch(tmp_dir).map do |path|
     grammar = load_grammar(path)
-    scope = grammar['scopeName']
+    scope = grammar['scopeName'] || grammar['scope']
```

Comment:
In YAML grammar files, the scope is under the `scope` key -- as opposed to the usual `scopeName` key.